Program.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace WordAnalysis
+{
+    class Program
+    {
+        private static readonly string[] Separators = { ".", ",", "-", "?", "!", ";", ":", "\"", "(", ")", "—" , "”", "“", "‘"};
+        
+        static void Main(string[] args)
+        {
+            var pathForRecord = @"C:\Users\Мария\Desktop\WordAnalysis\writer.txt";
+            var pathForRead = @"C:\Users\Мария\Desktop\WordAnalysis\reader.txt";
+            var input = "";
+            var dict = new Dictionary<string, double>();
+
+            #region Чтение входных данных
+
+            try
+            {
+                using var sr = new StreamReader(pathForRead);
+                input = sr.ReadToEnd();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                throw;
+            }
+
+            #endregion
+
+            #region Обработка входных данных
+
+            var allText = input;
+            foreach (var separator in Separators)
+            {
+                allText = allText.Replace(separator, "");

```

Comment:
Наверное, здесь нужно
```suggestion
                allText = allText.Replace(separator, " ");

```